crates/teloxide/Cargo.toml

```diff
@@ -63,8 +63,8 @@ full = [
 
 
 [dependencies]
```

Comment:
you forgot to update the `package.version` field

Reply:
Fixed.